knowlg-automation/terraform/aws/eks.tf

```diff
@@ -0,0 +1,47 @@
+resource "aws_eks_cluster" "eks_cluster" {
+  name  = var.cluster_name
+  role_arn = aws_iam_role.eks.arn
+   vpc_config {
+    subnet_ids = [
+      aws_subnet.private-us-west-1a.id,
```

Comment:
Is there any specific reason why we choose `us-west-1` region?

Reply:
no there is no specific reason for the region as the user can also config the region based on the needs {ideology taken same as of the azure scripts}